.prettierrc.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   semi: true,
   singleQuote: true,
   trailingComma: 'es5',
+  endOfLine: 'auto',
```

Comment:
Why this change?

Reply:
I use Windows as an operating system. In my OS, by default, line breaks are set as CR LF. Prettier highlights the whole file as an error. This setting removes the error if the line breaks are not CRLF. Moreover, for git it does not matter at all.